mapstore/log4j.properties

```diff
@@ -12,4 +12,4 @@ log4j.logger.com.trg=INFO
 log4j.appender.fileAppender=org.apache.log4j.RollingFileAppender
 log4j.appender.fileAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.fileAppender.layout.ConversionPattern=%p   %d{yyyy-MM-dd HH:mm:ss.SSS}   %C{1}.%M() - %m %n
-log4j.appender.fileAppender.File=/tmp/mapstore.log
\ No newline at end of file
+log4j.appender.fileAppender.File=/tmp/mapstore.log
```

Comment:
`fileappender` can be removed in the docker context

Reply:
done!